### ISSUE TITLE
version 2.28.0 Cannot find modules codex-notifier,  codex-tooltip

### ISSUE THREAD

**Issue by MussaratAziz** [codex-team/editor.js#2462]
After update i am getting following errors

Error: node_modules/@editorjs/editorjs/types/api/notifier.d.ts:1:78 - error TS2307: Cannot find module 'codex-notifier' or its corresponding type declarations.

1 import {ConfirmNotifierOptions, NotifierOptions, PromptNotifierOptions} from 'codex-notifier';
                                                                               ~~~~~~~~~~~~~~~~
Error: node_modules/@editorjs/editorjs/types/api/tooltip.d.ts:4:46 - error TS2307: Cannot find module 'codex-tooltip' or its corresponding type declarations.

4 import {TooltipContent, TooltipOptions} from 'codex-tooltip';
                                               ~~~~~~~~~~~~~~~

Thanks

**Comment by mrvdot** [comment]
I was able to fix that by just manually installing them:

```
npm i -S codex-notifier codex-tooltip
```

Looks like they were listed as dev dependencies, so they aren't installed by default anymore

**Comment by MussaratAziz** [comment]
> I was able to fix that by just manually installing them:
> 
> ```
> npm i -S codex-notifier codex-tooltip
> ```
> 
> Looks like they were listed as dev dependencies, so they aren't installed by default anymore

Thank you @mrvdot

**Comment by Fiehra** [comment]
installing the packages in my project did not fix the error for me. it is still crying about the missing modules. am i missing something? i just added the editorJS package and it didn work out of the box. also it looks to me like its missing type definition files

**Comment by MussaratAziz** [comment]
i dont know about your installed packages but i have some thing like this  i am using 2.29 rc its similar for 2.28.0
"dependencies":{
 "@editorjs/editorjs": "^2.29.0-rc.0",
        "codex-notifier": "^1.1.2",
        "codex-tooltip": "^1.0.5"
} and editorjs is displayed in my container. Hope it helps

**Comment by neSpecc** [comment]
Resolved by #2842
